datafusion/core/src/execution/runtime_env.rs

```diff
@@ -172,9 +155,9 @@ impl RuntimeConfig {
         self
     }
 
-    /// Customize memory manager
-    pub fn with_memory_manager(mut self, memory_manager: MemoryManagerConfig) -> Self {
-        self.memory_manager = memory_manager;
+    /// Customize memory policy
+    pub fn with_memory_policy(mut self, memory_pool: Arc<dyn MemoryPool>) -> Self {
```

Comment:
We could eliminate the word `policy` and explain that different pool comes with different allocation policies.

Reply:
Nice catch, this was the old name :+1: